apps/src/javalab/Javalab.js

```diff
@@ -181,7 +182,7 @@ Javalab.prototype.init = function(config) {
     isSubmitted: !!config.level.submitted
   });
 
-  registerReducers({javalab});
+  registerReducers({javalab, playground});
```

Comment:
Should the playground reducers get registered in the Playground constructor? (Not a leading question - asking because I don't know). I'm wondering if doing so would prevent bloat in the redux store for non-playground levels.

Reply:
I'm not sure, for now the playground store for a non-playground level would only contain an empty object, so I'm not worried about bloat. This at least keeps all the javalab redux initialization together, which feels cleaner to me. If the playground store gets bigger I think it could make sense to move the initialization elsewhere.